app/src/main/java/com/example/apkproject/MainActivity.kt

```diff
@@ -33,6 +56,10 @@ class MainActivity : AppCompatActivity() {
                 override fun onSuccess(asyncActionToken: IMqttToken?) {
                     println("Connection successfull")
                     mqtt.subscribe("testtest", 1)
+
+                    val clientId = MqttClient.generateClientId()
+                    mqttClient = MqttAndroidClient(applicationContext,"tcp://deneme-234@broker.mqtt.cool:1883",clientId)
```

Comment:
Hardcoded string, you should to be create `Constants` and you need to create value for "tcp://deneme-234@broker.mqtt.cool:1883"